publish.js

```diff
@@ -0,0 +1,15 @@
+var ghpages = require('gh-pages');
+var path = require('path');
+
+ghpages.publish('public',{
+	user: {
+		name: 'OpenFisca-Bot',
+	},
+	branch: 'gh-pages',
+	message: 'Auto-commit from master branch'
+	},function(err) {
+		if(err){
+			console.error(err)
+    		process.exit(1)
```

Comment:
I'm not sure I see what you mean by “indentation for a function's argument”.
Indentation is leading space following a newline. A function argument is rarely the first token on a line.
If you mean, for example, the `err` in `function(err)`, then you mean “spacing“, not “indentation”  🙂

There is, as always for what is style, no definite standard. I recommend [Google's JavaScript Styleguide](https://google.github.io/styleguide/jsguide.html#formatting-function-arguments), but keep in mind that the only important thing is that your codebase is consistent.